certified-connectors/BuildingMindsTwin/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "required",
+          "scopes": [
+            "https://buildingminds.onmicrosoft.com/cd1b0fa3-bc0f-4cd7-a09c-dbaf6c19db40/user_impersonation openid offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "redirectUrl": "https://global-test.consent.azure-apim.net/redirect",
```

Comment:
@duncanbinger You need to provide only one 'redirectURL' and that should not be test one.

Reply:
@sriyen-msft - I have removed the test redirect URI.